src/carousel/carousel.ts

```diff
@@ -267,6 +306,14 @@ export interface NgbSlideEvent {
    * Possible values are `'left' | 'right'`.
    */
   direction: NgbSlideEventDirection;
+
+  /**
+   * Whether the pause() method was called (and no cycle() call was done afterwards).
+   */
+  pause: boolean;
+
+  /** Source triggering the slide change event */
```

Comment:
I would make the comment a bit more explicative. It's our only way to document `NgbSlideEventSource` as we don't take into account enum in the doc generator.